Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -763,6 +763,7 @@ Once you identify a list of IPC mechanisms, review the source code to see whethe
 
 In the following, we use two example apps and give examples of identifying vulnerable IPC components:
 
+- ["OVAA"](https://github.com/oversecured/ovaa "Oversecured Vulnerable Android App")
```

Comment:
The Oversecured app should be added to the vulnerable app section instead: https://github.com/OWASP/owasp-mstg/blob/master/Document/0x08-Testing-Tools.md#vulnerable-applications